templates/layout.html

```diff
@@ -30,6 +30,13 @@ <h1 class="header__title">Partner.org</h1>
 
   {% block script %}
   <script>
+    {% if hypothesis_api_url %}
+      {% autoescape false %}
+        var hypothesisApiUrl = "{{ hypothesis_api_url }}";
+      {% endautoescape %}
+    {% else %}
+      var hypothesisApiUrl = null;
+    {% endif %}
```

Comment:
Since `hypothesis_api_url` is required, I don't think you need the if/else here, you can just assume that `hypothesis_api_url` exists